app/controllers/cwic_controller.rb

```diff
@@ -15,6 +15,12 @@ def download
     render content_type: 'text/html', layout: false
   end
 
+  def granule
```

Comment:
I'm not sure you needed a separate route for this. I think you could just use the `/cwic` route, but maybe I'm missing something. The separate route for `/cwic/edsc_download` allows the HTML rendering, but `/cwic` is just a pass-through.

If for some reason you *do* need a separate route, call it "edsc_granule" in the URL to avoid clashing with legitimate CWIC routes.